package.json

```diff
@@ -34,11 +34,11 @@
   },
   "dependencies": {
     "request-promise-core": "1.1.1",
-    "stealthy-require": "^1.1.0",
-    "tough-cookie": ">=2.3.3"
+    "stealthy-require": "^1.1.1",
+    "tough-cookie": "^2.5.0"
   },
   "peerDependencies": {
-    "request": "^2.34"
+    "request": "^2.88.0"
```

Comment:
@jasonmit if you update the peer dep here you will need to update the dev dep below too. but why did you update the peer dep in the first place? is it required for the fix to work? 🤔

Reply:
Good catch, removed!  (Left over fragment as I was debugging the broken CI pipeline)